tests/integration/interface_common_test.py

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2018-2019 Red Hat, Inc.
+#
+# This file is part of nmstate
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+
+import libnmstate
+
+from .testlib.statelib import INTERFACES
+
+
+def test_set_empty_iface_description(eth1_up):
```

Comment:
It will be more interesting to check its removal, correlating to the original issue found.
- add a non empty description.
- remove the description.

As these are integration tests which are slow (unlike unit tests) I would just merge these two tests you added into one, but still assert each step to check that the outcome is correct.